doc/manuals/admin/build_source.md

```diff
@@ -47,7 +47,7 @@ commands that require root privilege):
         wget http://ftp.gnu.org/gnu/libmicrohttpd/libmicrohttpd-0.9.48.tar.gz
         tar xvf libmicrohttpd-0.9.48.tar.gz
         cd libmicrohttpd-0.9.48
-        ./configure --disable-messages --disable-postprocessor --disable-dauth --disable-epoll
+        ./configure --disable-messages --disable-postprocessor --disable-dauth
```

Comment:
From https://www.gnu.org/software/libmicrohttpd/manual/html_node/microhttpd_002dintro.html:

> The highest performance can typically be obtained with a thread pool using epoll

So probably it is better not disabling epoll.

NTC (just informative)